modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -0,0 +1,602 @@
+            std::vector<int> selectedImagesVector;
+            const std::deque<int> selectedImages = pair->getSelectedImages();
+            std::for_each(selectedImages.begin(), selectedImages.end(), [&](int i) {
+                selectedImagesVector.push_back(i);
+                });
+                
+            glm::dvec2 spherical = pair->targetDirectionEquatorial();
+            glm::dvec3 cartesian = skybrowser::sphericalToCartesian(spherical);
+            
+                   
+            std::vector<double> cartesianVec = { 
+                cartesian.x, 
+                cartesian.y, 
+                cartesian.z 
+            };
+            // Convert color to vector so ghoul can read it
+            glm::ivec3 color = pair->borderColor();
+            std::vector<int> colorVec = { color.r, color.g, color.b };
+
+            // Convert color to vector so ghoul can read it
+            glm::vec2 size = pair->size();
+            std::vector<float> sizeVec = { size.x, size.y };
+
+            ghoul::lua::push(L, id);
+            lua_newtable(L);
+            // Push ("Key", value)
+            ghoul::lua::push(L, "id", id);
+            lua_settable(L, -3);
+            ghoul::lua::push(L, "name", pair->browserGuiName());
+            lua_settable(L, -3);
+            ghoul::lua::push(L, "FOV", pair->verticalFov());
+            lua_settable(L, -3);
+            ghoul::lua::push(L, "selectedImages", selectedImagesVector);
+            lua_settable(L, -3);
+            ghoul::lua::push(L, "cartesianDirection", cartesianVec);
+            lua_settable(L, -3);
+            ghoul::lua::push(L, "ra", spherical.x);
+            lua_settable(L, -3);
+            ghoul::lua::push(L, "dec", spherical.y);
+            lua_settable(L, -3);
+            ghoul::lua::push(L, "color", colorVec);
+            lua_settable(L, -3);
+            ghoul::lua::push(L, "isLocked", pair->isLocked());
+            lua_settable(L, -3);
+            ghoul::lua::push(L, "size", sizeVec);
+            lua_settable(L, -3);
+              
+            // Set table for the current target
+            lua_settable(L, -3);
+                
+        }
+    }
+
+    return 1;
+}
+int adjustCamera(lua_State* L) {
+	ghoul::lua::checkArgumentsAndThrow(L, 1, "lua::adjustCamera");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    if(module->isCameraInSolarSystem()) {
+        module->lookAtTarget(id);
+    }
+
+	return 0;
+}
+
+int setOpacityOfImageLayer(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, 3, "lua::setOpacityOfImageLayer");
+    const std::string id = ghoul::lua::value<std::string>(L, 1);
+    const int i = ghoul::lua::value<int>(L, 1);
+    double opacity = ghoul::lua::value<double>(L, 1);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+
+    if (module->getPair(id)) {
```

Comment:
Can store the result of the `getPair` function and don't have to access it again in the if statement